src/components/Searchbar/InstantSearch.tsx

```diff
@@ -19,39 +26,73 @@ export const InstantSearch = () => {
 
   const onLsChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
     setSearch(e.target.value);
+    setIsSearchLoading(true);
 
     await axios
-        .get(`https://jsonplaceholder.typicode.com/albums`)
-        .then((res) => {
-          setPosts(res.data);
-        })
-        .catch((error) => {
-          if (axios.isCancel(error) || error) {
-            console.log('Could not get');
-          }
-        });
+      .get(`https://jsonplaceholder.typicode.com/albums`)
+      .then((res) => {
+        setPosts(res.data);
+        setIsSearchLoading(false);
+      })
+      .catch((error) => {
```

Comment:
If there are problems with api we will have infinity loading, so you can add here set loading to false also